src/components/styles/sprite-canvas.css.ts

```diff
@@ -0,0 +1,11 @@
+import { style } from "@vanilla-extract/css";
+
+export const spriteCanvasStyle = style({
+  height: `auto`,
+  background: `#aaaaaa`,
+  display: `flex`,
+  flexDirection: "column",
+  alignItems: "center",
+  padding: "1rem",
+  borderRadius: "10px",
```

Comment:
Might be worth adding some rules to the deno linter. Being consistent in quotation ticks isn't all that important, but it could be confusing as more people look at the code.